src/main/java/com/amazon/ion/impl/IonReaderContinuableCoreBinary.java

```diff
@@ -1398,149 +1404,13 @@ private enum State {
     // The current state.
     private State state = State.READING_VALUE;
 
-    /**
-     * Eagerly collects the annotations on the current value.
-     * @return the annotations, or an empty list if there are none.
-     */
-    private List<SymbolToken> getAnnotations() {
-        if (!hasAnnotations) {
-            return Collections.emptyList();
-        }
-        List<SymbolToken> out = new ArrayList<>();
-        consumeAnnotationTokens(out::add);
-        return out;
-    }
-
     /**
      * Reads macro invocation arguments as expressions and feeds them to the MacroEvaluator.
-     * TODO ideally this could be factored out of IonReaderContinuableCoreBinary.
-     *  See {@link com.amazon.ion.impl.macro.MacroExpressionizer}. Currently, this relies on class internals.
      */
-    class EExpressionArgsReader {
```

Comment:
The red in here moved into the top-level `EExpressionArgsReader`, shared by the text reader.